scripts/netContain/scripts/contivNet.sh

```diff
@@ -114,34 +116,39 @@ if [ "$plugin" == "kubernetes" ]; then
    fi
 fi
 
+set +eo pipefail
```

Comment:
Worst case, turn set `+e` before the /contiv/bin/netmaster and /contiv/bin/netplugin calls and turn it on immediately afterward.  You could also leave `-e` enabled and just add `|| true` after the commands to ignore their exit codes when they crash.

There's nothing being piped, so there's no need to turn off `-o pipefail`